src/patternfly/components/ContextSelector/examples/context-selector-example.hbs

```diff
@@ -1,5 +1,5 @@
-{{#> context-selector id="context-selector-collapsed-example" context-selector--label-text="Selected Project"}}
-  {{#> context-selector-toggle}}
+{{#> context-selector context-selector--id="context-selector-collapsed-example" context-selector--label-text="Selected Project"}}
+  {{#> context-selector-toggle context-selector-toggle--attribute=(concat 'id="' context-selector--id '-toggle"' 'aria-labelledby="' context-selector--id '-toggle ' context-selector--id '-label"')}}
```

Comment:
Can you switch the order in the `aria-labelledby` value so that it's: 
`'aria-labelledby="' context-selector--id '-label ' context-selector--id '-toggle"`

This will announce the label text "Selected Project:" first before announcing the toggle label (i.e. the selected project)